.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - "6"
+  - "4"
```

Comment:
Do we really want to support this? Like, really? This isn't a node module, it's a browser thing so testing against ancient node versions isn't really helping the project here. It only causing the tests to run longer.